docs/_templates/snippets.html

```diff
@@ -13,7 +13,7 @@
         {% set snippet_file = "../../src/" + component_name + "/snippets/" + snippet.file %}
         {% endif %}
         <div class="snippet-demo">
-          <div class="snippet-demo-container">
+          <div class="snippet-demo-container demo-{{component_name}} demo-{{component_name}}--{{snippet.file | replace('.html', '')}}">
```

Comment:
Nit: the snippet class would ideally be `demo-{{component_name}} demo-{{component_name}}__{{snippet.file | replace('.html', '')}}` (underscores instead of dashes), but don't worry about it if it would mean changing a lot of stuff.